libs/scripts/taxonomy.js

```diff
@@ -67,7 +67,7 @@ async function fetchTaxonomy(target) {
   return fetch(target).then((response) => response.json());
 }
 
-function parseTaxonomyJson(data, root) {
+function parseTaxonomyJson(data, root, route) {
```

Comment:
I see that usually the `taxonomyRoot` defaults to `/topics` before this method is being used. Would it make sense to have that as the default value for `route` here as well?